app/Spago.hs

```diff
@@ -250,21 +281,55 @@ listPackages = do
 sources :: IO ()
 sources = do
   config <- Config.ensureConfig
-  let
-    deps = getAllDependencies config
-    globs = getGlobs deps
-  _ <- traverse echo globs
+  deps <- getProjectDeps config
+  _ <- traverse echo $ getGlobs deps
   pure ()
 
 
+verify :: Maybe Int -> Maybe PackageName -> IO ()
+verify maybeLimit maybePackage = do
```

Comment:
below comment would remove the need for Maybe whatever

Reply:
I left the `Maybe` in, so we can contain all the functionality (i.e. for both `verify` and `verify-set`) in this `verify` block